package.json

```diff
@@ -98,15 +99,15 @@
     "@craco/craco": "^6.4.5",
     "@emotion/babel-plugin": "^11.10.6",
     "@sentry/webpack-plugin": "^1.20.0",
+    "@testing-library/jest-dom": "^5.16.4",
```

Comment:
pls make sure not to commit formatting/ordering changes together with business logic ones